src/renderer/components/dock/dock.tsx

```diff
@@ -46,19 +46,31 @@ interface Props {
 
 @observer
 export class Dock extends React.Component<Props> {
-  onKeydown = (evt: React.KeyboardEvent<HTMLElement>) => {
-    const { close, closeTab, selectedTab } = dockStore;
+  private element = React.createRef<HTMLDivElement>();
 
-    if (!selectedTab) return;
-    const { code, ctrlKey, shiftKey } = evt.nativeEvent;
+  componentDidMount() {
+    document.addEventListener("keydown", this.onKeyDown);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener("keydown", this.onKeyDown);
+  }
```

Comment:
Listening `keydown` globally allowing to close dock tabs in quick sequence, by repeating pressing `cmd+w`. Before the change it was impossible since the focus was jumped after closing each tab, so the event was no longer listened to.